mkt/extensions/models.py

```diff
@@ -52,10 +53,11 @@ def public(self):
 
 class Extension(ModelBase):
     # Automatically handled fields.
-    uuid = UUIDField(auto=True)
+    last_updated = models.DateTimeField(db_index=True, null=True)
```

Comment:
We don't, because it's only set when a new version is `*published*`. We don't want it to be updated on other saves.